doc/classes/GPUParticles2D.xml

```diff
@@ -41,8 +41,16 @@
 				[b]Note:[/b] [method emit_particle] is only supported on the Forward+ and Mobile rendering methods, not Compatibility.
 			</description>
 		</method>
+		<method name="request_particles_process">
+			<return type="void" />
+			<param index="0" name="process_time" type="float" />
+			<description>
+				Requests the particles to process for extra process time during a single frame. Used for seeking in particles playback.
```

Comment:
```suggestion
				Requests the particles to process for extra process time during a single frame. Used for seeking in particles playback.
				[b]Note:[/b] To get consistent results when seeking particles back and forth, [member seed_deterministic] must be [code]false[/code]. Otherwise, every call to [method request_particles_process] will lead to an entirely different visual outcome.
```